packages/trader/src/Stores/useTraderStores.tsx

```diff
@@ -23,14 +29,54 @@ type TToastBoxObject = {
 type TOverrideTradeStore = Omit<
     TradeStore,
     | 'accumulator_range_list'
+    | 'barrier_count'
```

Comment:
i think we are only supposed to add in TOverrideTradeStore only if typescript can't infer type. Otherwise, we just type everything as TTradeStore instead of TOverrideTradeStore. Can you please check if type can be inferred before adding to this list.

Reply:
@henry-deriv totally agree👍🏻thank you for pointing this out!